.github/workflows/nightly-publish-to-npm.yml

```diff
@@ -1,12 +1,29 @@
 name: Nightly Publish Package to NPM
 on:
   schedule:
-    - cron: '0 5 * * *'
+    - cron: '0 5 * * 1-6'

```

Comment:
So.. This means 5:00 on Monday-Saturday? I'd prefer 6:00 on Monday-Friday, if that's not too much to ask :P